pom.xml

```diff
@@ -66,6 +68,14 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
+        <configuration>
```

Comment:
The default threshold in plugin-pom is `Medium`, hence it misses the File leak on exceptional path in this case. Ideally should be fixed in Parent POM, of course